src/docs/_components/Placeholder/Placeholder.jsx

```diff
@@ -5,12 +5,14 @@ import styles from './Placeholder.scss';
 export const Placeholder = ({
```

Comment:
On second thought, I don't think the name `priority` is good to indicate `filled` vs `outline`.

Reply:
I agree we should avoid using props with same name with for different purpose. They might be cases when we are unable to use unique name but I hope this is not the case.

However I am not sure that we want to use full set of `priority` as it is for `variant`. `variant` is simple coloring of component, `priority` is another level of visual difference. In my opinion. And therefore it is not necessary to use all of them. But again, I believe in Adam's decision .

Reply:
> Namely we should rename the variant prop in SelectField TextField and TextArea to priority. This would make it compatible with Button, ButtonGroup and Badge.

Nice touch. Problem is that `variant` of a form field is not the same as `priority` of other components (`Badge`, `Button`). While priority of `Badge`s and `Button`s actually tells a lot of component's _visual priority_ (making it more or less pop out from its context), form field variants really are just _visual alternatives,_ all equal one to each other from visual priority perspective. That brings us to the root of this confusion. While the _property_ means something different, the available _values_ are the same: `filled` and `outline`:

<img width="852" alt="obrazek" src="https://user-images.githubusercontent.com/5614085/121226815-3d563000-c88b-11eb-81bb-a92f537cb807.png">

To avoid this confusion, we may want to rename the values like this:

- (on the left) `outline` to `bordered` (maybe something better? See below),
- (on the right) `filled` to `bottom-line`.

I wouldn't go for `border-all` and `border-bottom` (or similar) because the difference is in available styling options and in the focus effect rather than just the number of bordered sides.

Also, we may object it may be confusing to call the variant `bordered` as the border can be removed with custom styling (theme). But I haven't come across any better name than `default` which is too vague.

Reply:
Possible option: make the first variant primary (default) and the other optional (`bottomLine`, boolean). Don't close the door to floating labels that may come in future.

Reply:
Having changed the `variant` prop to `color`, I think this is less of a problem now. It may be still confusing that `filled` and `outline` are values used for both `priority` and `variant` props, but I don't see it as a major problem.